File: glean/lang/java-alpha/indexer/examples/java/com/facebook/glean/glean/Glean.java
// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.

package com.facebook.glean.glean;

/*
  Comment for Glean class
*/
@Deprecated
public class Glean {

  public static final String LOG_TAG = "Glean";

  @Deprecated
  @SuppressWarnings("deprecation")
  public POJO fetch() {
    return new POJO();
  }

  public static void main(String[] args) {
    Glean glean = new Glean();
    POJO pojo = glean.fetch();
    pojo.getE();
    pojo.getE(10);
    pojo.getList(null);
  }
}

File: glean/lang/kotlin/indexer/examples/java/com/facebook/glean/glean/Glean.kt
// (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.

package com.facebook.glean.glean

open class SuperGlean {}

class Glean : SuperGlean {

  constructor() : super()

  fun fetch(): ClassA {
    return ClassA()
  }

  companion object {
    const val LOG_TAG = "Glean"

    @JvmStatic
    fun main(args: Array<String>) {
      val glean = Glean()
      val instance = glean.fetch()
      instance.e
      instance.getE(10)
      instance.getList(null)
    }
  }
}
